PureBasicIDE/ProcedureBrowser.pb

```diff
@@ -461,7 +475,7 @@ Procedure ProcedureBrowser_EventHandler(*Entry.ToolsPanelEntry, EventGadgetID)
         ; Unfold the procedure block if it was folded
         SendEditorMessage(#SCI_ENSUREVISIBLE, ProcedureList()\Line)
         
-        CompilerIf #CompileMac
+        CompilerIf #CompileMac Or #CompileLinux ;- Added by mk-soft; Date 22.05.2020
```

Comment:
This comment is no longer necessary because Git stores this information about who added, changed, and removed which line.